### PR TITLE
Notify project manager about projects violating lease stacking policy

### PR DESCRIPTION
**Pull request by AnishReddyRavula**
Replaces lease stacking reaper with the notifier
We do not want to delete users' lease automatically if they are violating a policy. Inform project manager about the violation and letting them take a decision is a valid thing to do.

This goes through blazar leases for cases of violations of lease stacking and notifies them to project manager

This uses python-chi instead of osapi as it causing issues with authentication with openrc

### CODE REVIEW

**Review by Mark-Powers — APPROVED** [review]
The code looks good to me. Could you delete the old script from this repo as well so we don't accidentally use it in the future?

File: hammers/scripts/tests/lease-stacking-test-config.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        "openstack"
+    ],
+    "lease_coverage_threshold": 0.9,
+    "high_end_node_coverage_threshold": 0.5,
+    "minimum_lease_window_days": 21,
+    "min_nodes_for_coverage": 4,
+    "exclude_node_types": [],
+    "high_end_node_types": [],
+    "sender_email": "noreply@chameleoncloud.org",
+    "manager_email": "help@chameleoncloud.org"
```
Comment (msherman64) [inline hammers/scripts/tests/lease-stacking-test-config.json:14]: We shouldn't send to "real" emails during the test!
(just to prevent accidents)

**Review by msherman64 — APPROVED** [review]
Looks good to me, just a request to change the default test email for safety.

**Comment by msherman64** [comment]
A final note.

When we deploy this, we'll need to manually go and remove the old system timers and units. Ansible will not remove them automatically.

**Pull request merged**